src/gg.regression.unity.bots/Runtime/Scripts/RGLegacyInputUtility/RGLegacyInputWrapper.cs

```diff
@@ -320,64 +329,85 @@ public static void Update()
                         state.rawValue *= -1.0f;
                     }
                     state.smoothedValue = state.rawValue; // smoothing is not applied on mouse movement
-                } 
+                }
                 else if (entry.type == InputManagerEntryType.JOYSTICK_AXIS)
                 {
                     // joysticks unsupported currently
                     // eventually this will read the joystick value, incorporating entry.dead for dead zone
-                } 
+                }
                 else
                 {
                     RGDebug.LogWarning($"Unexpected input manager entry type: {entry.type}");
                 }
             }
-                
+
             // Invoke the appropriate mouse event handlers on MonoBehaviours
             {
-                (GameObject, Camera)? hitObject = null;
-                (GameObject, Camera)? hitObject2D = null;
                 int numCameras = Camera.allCamerasCount;
                 if (_camerasBuf == null || _camerasBuf.Length != numCameras)
                 {
                     _camerasBuf = new Camera[numCameras];
                 }
+
                 Camera.GetAllCameras(_camerasBuf);
                 foreach (Camera camera in _camerasBuf)
                 {
                     if (camera == null || camera.eventMask == 0 || camera.targetTexture != null)
                     {
                         continue;
                     }
-                        
-                    int cameraRaycastMask = camera.cullingMask & camera.eventMask;
-                        
+
+                    var cameraRaycastMask = camera.cullingMask & camera.eventMask;
+
+                    (GameObject, GameObject) result = (null, null);
```

Comment:
flagging a `The variable 'result' is assigned but its value is never used` warning here.